lib/db/leveldb_dbinstance.go

```diff
@@ -306,75 +319,53 @@ func (db *Instance) getGlobal(folder, file []byte, truncate bool) (FileIntf, boo
 		return nil, false
 	}
 
-	var vl VersionList
-	err = vl.Unmarshal(bs)
-	if err == leveldb.ErrNotFound {
-		return nil, false
-	}
```

Comment:
I wonder why this error check is here. Seems like an unprobable error to be returned by Unmarshal?